{
  "uid" : "b203e98fbf94a4d1",
  "name" : "test_get_css_property_header_auth_signup[font-size-18px-\\u0440\\u0430\\u0437\\u043c\\u0435\\u0440\\u0430 \\u0448\\u0440\\u0438\\u0444\\u0442\\u0430-https://pulsewave.ru/auth/password?email=-chrome]",
  "fullName" : "tests.test_header.test_header.TestHeader#test_get_css_property_header_auth_signup",
  "historyId" : "c8dbf1e8a628678f771aaef4fb3b3c5a",
  "time" : {
    "start" : 1714443582540,
    "stop" : 1714443582540,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome-headless-shell=124.0.6367.60)\nStacktrace:\n#0 0x5599a296be33 <unknown>\n#1 0x5599a265b567 <unknown>\n#2 0x5599a2653879 <unknown>\n#3 0x5599a2643852 <unknown>\n#4 0x5599a2644e44 <unknown>\n#5 0x5599a2643bbe <unknown>\n#6 0x5599a26432e0 <unknown>\n#7 0x5599a2643181 <unknown>\n#8 0x5599a26410ec <unknown>\n#9 0x5599a264160a <unknown>\n#10 0x5599a265deef <unknown>\n#11 0x5599a26e7695 <unknown>\n#12 0x5599a26c8362 <unknown>\n#13 0x5599a26e6ae2 <unknown>\n#14 0x5599a26c8103 <unknown>\n#15 0x5599a26991ba <unknown>\n#16 0x5599a2699b8e <unknown>\n#17 0x5599a293059b <unknown>\n#18 0x5599a293434b <unknown>\n#19 0x5599a291c681 <unknown>\n#20 0x5599a2934eb2 <unknown>\n#21 0x5599a29016af <unknown>\n#22 0x5599a295b428 <unknown>\n#23 0x5599a295b600 <unknown>\n#24 0x5599a296af84 <unknown>\n#25 0x7f8132094ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"0e6caf41473bd334c99c1d7cc0569ee7\")>\nurl = 'https://pulsewave.ru/auth/password?email='\n\n    @pytest.fixture()\n    def header_open(driver, url):\n        page = HeaderPage(driver)\n>       driver.get(url)\n\ntests/test_header/conftest.py:8: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:356: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:347: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f17c7c057f0>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x5599a295b600 \\\\u003Cunknown>\\\\n#24 0x5599a296af84 \\\\u003Cunknown>\\\\n#25 0x7f8132094ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome-headless-shell=124.0.6367.60)\nE       Stacktrace:\nE       #0 0x5599a296be33 <unknown>\nE       #1 0x5599a265b567 <unknown>\nE       #2 0x5599a2653879 <unknown>\nE       #3 0x5599a2643852 <unknown>\nE       #4 0x5599a2644e44 <unknown>\nE       #5 0x5599a2643bbe <unknown>\nE       #6 0x5599a26432e0 <unknown>\nE       #7 0x5599a2643181 <unknown>\nE       #8 0x5599a26410ec <unknown>\nE       #9 0x5599a264160a <unknown>\nE       #10 0x5599a265deef <unknown>\nE       #11 0x5599a26e7695 <unknown>\nE       #12 0x5599a26c8362 <unknown>\nE       #13 0x5599a26e6ae2 <unknown>\nE       #14 0x5599a26c8103 <unknown>\nE       #15 0x5599a26991ba <unknown>\nE       #16 0x5599a2699b8e <unknown>\nE       #17 0x5599a293059b <unknown>\nE       #18 0x5599a293434b <unknown>\nE       #19 0x5599a291c681 <unknown>\nE       #20 0x5599a2934eb2 <unknown>\nE       #21 0x5599a29016af <unknown>\nE       #22 0x5599a295b428 <unknown>\nE       #23 0x5599a295b600 <unknown>\nE       #24 0x5599a296af84 <unknown>\nE       #25 0x7f8132094ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1714443483373,
      "stop" : 1714443483373,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "_session_faker",
    "time" : {
      "start" : 1714443483355,
      "stop" : 1714443483372,
      "duration" : 17
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1714443582540,
      "stop" : 1714443582739,
      "duration" : 199
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "header_open",
    "time" : {
      "start" : 1714443582740,
      "stop" : 1714443582958,
      "duration" : 218
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome-headless-shell=124.0.6367.60)\nStacktrace:\n#0 0x5599a296be33 <unknown>\n#1 0x5599a265b567 <unknown>\n#2 0x5599a2653879 <unknown>\n#3 0x5599a2643852 <unknown>\n#4 0x5599a2644e44 <unknown>\n#5 0x5599a2643bbe <unknown>\n#6 0x5599a26432e0 <unknown>\n#7 0x5599a2643181 <unknown>\n#8 0x5599a26410ec <unknown>\n#9 0x5599a264160a <unknown>\n#10 0x5599a265deef <unknown>\n#11 0x5599a26e7695 <unknown>\n#12 0x5599a26c8362 <unknown>\n#13 0x5599a26e6ae2 <unknown>\n#14 0x5599a26c8103 <unknown>\n#15 0x5599a26991ba <unknown>\n#16 0x5599a2699b8e <unknown>\n#17 0x5599a293059b <unknown>\n#18 0x5599a293434b <unknown>\n#19 0x5599a291c681 <unknown>\n#20 0x5599a2934eb2 <unknown>\n#21 0x5599a29016af <unknown>\n#22 0x5599a295b428 <unknown>\n#23 0x5599a295b600 <unknown>\n#24 0x5599a296af84 <unknown>\n#25 0x7f8132094ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 917, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/PulseWave/PulseWave/tests/test_header/conftest.py\", line 8, in header_open\n    driver.get(url)\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 356, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.3/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1714443582991,
      "stop" : 1714443583049,
      "duration" : 58
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Тестирование Хедера"
  }, {
    "name" : "tag",
    "value" : "regress"
  }, {
    "name" : "parentSuite",
    "value" : "tests.test_header"
  }, {
    "name" : "suite",
    "value" : "test_header"
  }, {
    "name" : "subSuite",
    "value" : "TestHeader"
  }, {
    "name" : "host",
    "value" : "fv-az1385-609"
  }, {
    "name" : "thread",
    "value" : "2312-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.test_header.test_header"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "browser",
    "value" : "'chrome'"
  }, {
    "name" : "css_property",
    "value" : "'font-size'"
  }, {
    "name" : "figma",
    "value" : "'18px'"
  }, {
    "name" : "name",
    "value" : "'размера шрифта'"
  }, {
    "name" : "url",
    "value" : "'https://pulsewave.ru/auth/password?email='"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 3,
        "skipped" : 0,
        "passed" : 11,
        "unknown" : 0,
        "total" : 14
      },
      "items" : [ {
        "uid" : "884647ccd87bc808",
        "reportUrl" : "https://Nat754.github.io/PulseWave/9//#testresult/884647ccd87bc808",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome-headless-shell=124.0.6367.60)\nStacktrace:\n#0 0x5640d3348e33 <unknown>\n#1 0x5640d3038567 <unknown>\n#2 0x5640d3030879 <unknown>\n#3 0x5640d3020852 <unknown>\n#4 0x5640d3021e44 <unknown>\n#5 0x5640d3020bbe <unknown>\n#6 0x5640d30202e0 <unknown>\n#7 0x5640d3020181 <unknown>\n#8 0x5640d301e0ec <unknown>\n#9 0x5640d301e60a <unknown>\n#10 0x5640d303aeef <unknown>\n#11 0x5640d30c4695 <unknown>\n#12 0x5640d30a5362 <unknown>\n#13 0x5640d30c3ae2 <unknown>\n#14 0x5640d30a5103 <unknown>\n#15 0x5640d30761ba <unknown>\n#16 0x5640d3076b8e <unknown>\n#17 0x5640d330d59b <unknown>\n#18 0x5640d331134b <unknown>\n#19 0x5640d32f9681 <unknown>\n#20 0x5640d3311eb2 <unknown>\n#21 0x5640d32de6af <unknown>\n#22 0x5640d3338428 <unknown>\n#23 0x5640d3338600 <unknown>\n#24 0x5640d3347f84 <unknown>\n#25 0x7f18c8294ac3 <unknown>",
        "time" : {
          "start" : 1714357237984,
          "stop" : 1714357237984,
          "duration" : 0
        }
      }, {
        "uid" : "b499f58cef8d7820",
        "reportUrl" : "https://Nat754.github.io/PulseWave/8//#testresult/b499f58cef8d7820",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x564c2ed93e33 <unknown>\n#1 0x564c2ea83567 <unknown>\n#2 0x564c2eace27d <unknown>\n#3 0x564c2eace331 <unknown>\n#4 0x564c2eb11624 <unknown>\n#5 0x564c2eaf038d <unknown>\n#6 0x564c2eb0eae2 <unknown>\n#7 0x564c2eaf0103 <unknown>\n#8 0x564c2eac11ba <unknown>\n#9 0x564c2eac1b8e <unknown>\n#10 0x564c2ed5859b <unknown>\n#11 0x564c2ed5c34b <unknown>\n#12 0x564c2ed44681 <unknown>\n#13 0x564c2ed5ceb2 <unknown>\n#14 0x564c2ed296af <unknown>\n#15 0x564c2ed83428 <unknown>\n#16 0x564c2ed83600 <unknown>\n#17 0x564c2ed92f84 <unknown>\n#18 0x7f9a23a94ac3 <unknown>",
        "time" : {
          "start" : 1714272716632,
          "stop" : 1714272716632,
          "duration" : 0
        }
      }, {
        "uid" : "58111f9866820364",
        "reportUrl" : "https://Nat754.github.io/PulseWave/7//#testresult/58111f9866820364",
        "status" : "passed",
        "time" : {
          "start" : 1714184991999,
          "stop" : 1714184992038,
          "duration" : 39
        }
      }, {
        "uid" : "41ec14f8eb3563",
        "reportUrl" : "https://Nat754.github.io/PulseWave/6//#testresult/41ec14f8eb3563",
        "status" : "passed",
        "time" : {
          "start" : 1714148459511,
          "stop" : 1714148459547,
          "duration" : 36
        }
      }, {
        "uid" : "797ac289fb03c3d5",
        "reportUrl" : "https://Nat754.github.io/PulseWave/6//#testresult/797ac289fb03c3d5",
        "status" : "passed",
        "time" : {
          "start" : 1714098509683,
          "stop" : 1714098509718,
          "duration" : 35
        }
      }, {
        "uid" : "1e9ef1ee074cd24f",
        "reportUrl" : "https://Nat754.github.io/PulseWave/5//#testresult/1e9ef1ee074cd24f",
        "status" : "passed",
        "time" : {
          "start" : 1714014390377,
          "stop" : 1714014390411,
          "duration" : 34
        }
      }, {
        "uid" : "3e3f0789741371b5",
        "reportUrl" : "https://Nat754.github.io/PulseWave/211//#testresult/3e3f0789741371b5",
        "status" : "passed",
        "time" : {
          "start" : 1713945885703,
          "stop" : 1713945885738,
          "duration" : 35
        }
      }, {
        "uid" : "ffe649251bc6becb",
        "reportUrl" : "https://Nat754.github.io/PulseWave/4//#testresult/ffe649251bc6becb",
        "status" : "passed",
        "time" : {
          "start" : 1713925678577,
          "stop" : 1713925678610,
          "duration" : 33
        }
      }, {
        "uid" : "967c73820dcc363d",
        "reportUrl" : "https://Nat754.github.io/PulseWave/211//#testresult/967c73820dcc363d",
        "status" : "passed",
        "time" : {
          "start" : 1713901532555,
          "stop" : 1713901532589,
          "duration" : 34
        }
      }, {
        "uid" : "31b08ae95c1d6f11",
        "reportUrl" : "https://Nat754.github.io/PulseWave/210//#testresult/31b08ae95c1d6f11",
        "status" : "passed",
        "time" : {
          "start" : 1713885730792,
          "stop" : 1713885730831,
          "duration" : 39
        }
      }, {
        "uid" : "d9ae3881bad07eb7",
        "reportUrl" : "https://Nat754.github.io/PulseWave/209//#testresult/d9ae3881bad07eb7",
        "status" : "passed",
        "time" : {
          "start" : 1713883627103,
          "stop" : 1713883627136,
          "duration" : 33
        }
      }, {
        "uid" : "d643512611a25935",
        "reportUrl" : "https://Nat754.github.io/PulseWave/209//#testresult/d643512611a25935",
        "status" : "passed",
        "time" : {
          "start" : 1713882146584,
          "stop" : 1713882146617,
          "duration" : 33
        }
      }, {
        "uid" : "a471feb2272021ae",
        "reportUrl" : "https://Nat754.github.io/PulseWave/208//#testresult/a471feb2272021ae",
        "status" : "passed",
        "time" : {
          "start" : 1713879325419,
          "stop" : 1713879325457,
          "duration" : 38
        }
      } ]
    },
    "tags" : [ "regress" ]
  },
  "source" : "b203e98fbf94a4d1.json",
  "parameterValues" : [ "'chrome'", "'font-size'", "'18px'", "'размера шрифта'", "'https://pulsewave.ru/auth/password?email='" ]
}